lib/search_vector.rb

```diff
@@ -7,8 +7,9 @@ module SearchVector
   def self.query(query_string, expr=nil)
     (expr || Qexpr.new).where(
       "(supporters.fts @@ websearch_to_tsquery('english', $search)
-      OR donations.fts  @@ websearch_to_tsquery('english', $search))",
-      { search: query_string }
+      OR donations.fts  @@ websearch_to_tsquery('english', $search)
+      OR payments.id::varchar(255) = $search)",
+      { search: query_string.to_s }
```

Comment:
Why the change here?

Reply:
What change do you mean?

Reply:
You added a `.to_s` to `search_query`

Reply:
I had a problem passing an integer to `websearch_to_tsquery`